frontend/src/containers/Main/index.tsx

```diff
@@ -4,6 +4,7 @@ import Wrapper from '../../components/Wrapper'
 import Card from '../../components/Card'
 import { Header, Subheader, Paragraph } from '../../components/Typography'
 import TaskList from '../../components/TaskLisk'
```

Comment:
Let's convert `TaskList` from a styled component into a full-fledged React component. `TaskList` should import `TaskCard` and use it as shown on master. Make sure to pull from master before making these changes